src/pages/sessions/running.js

```diff
@@ -29,13 +34,74 @@ class Running extends Component {
 
     return (
       <TeacherLayout intl={intl} navbar={navbarConfig} sidebar={{ activeItem: 'runningSession' }}>
-        <Grid padded columns="2">
-          <Grid.Column>blebleble</Grid.Column>
-          <Grid.Column>blablabla</Grid.Column>
-        </Grid>
+        <div className="runningSession">
+          <div className="sessionProgress"><SessionProgress intl={intl} /></div>
+          <div className="feedback">
```

Comment:
Do you mean runningSession? Where else can we define ex. the margin for the complete runningSession container?